imgProc_sm.m

```diff
@@ -155,9 +155,17 @@ function imgProc_sm(infile, outfile, probename, n, nEvery, projectname, varargin
 warning off all
 diode_stats = zeros(1,diodenum);
 
-if strcmp(projectname, 'PECAN')  % For example for PECAN dataset 
-    disp('Testing...')  %% Add project specific code if you like
-end
+%if strcmp(projectname, 'PECAN')  % For example for PECAN dataset 
+%    if strcmp(probename, 'PIP')
+%        TimeOffsetSec = -12*3600.0;
```

Comment:
Were these commented lines in use when you were testing different correction techniques? I wasn't sure if they should still remain in the submitted code.